src/components/EventBox.jsx

```diff
@@ -21,15 +21,16 @@ const Card = styled.div`
   padding: 20px 30px;
 `
 
-export default function EventBox({ title, dateString, body, imgSrc, imgAlt }) {
+export default function EventBox({ title, dateString, body, imgSrc, imgAlt, footer, link }) {
   return (
     <Container>
       <Image src={imgSrc} alt={imgAlt} />
       <Card>
-        <Header3>{title}</Header3>
+        <Header3 as="a" href={link}>{title}</Header3>
```

Comment:
Can you add a little more padding underneath the header? With the underline, Header3 and Header4 seem a little squished. If Header3 isn't used anywhere else you could add it in the component itself, or you can add a wrapper around Header3 in this file. Something like 24px might make it look less squished.